shared-package.props

```diff
@@ -52,7 +51,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(SourceLinkGitHubVersion)" PrivateAssets="All" />
```

Comment:
This doesn't look right. Did you test SourceLink actually works?

Reply:
From [the link in the PR description](https://github.com/dotnet/sourcelink?tab=readme-ov-file#using-source-link-in-net-projects):

> If your project uses .NET SDK 8+ and is hosted by the above providers it does not need to reference any Source Link packages or set any build properties to include source control information in the PDB.